spec/dnsimple/client/services_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+
+describe Dnsimple::Client, ".services" do
+
+  subject { described_class.new(base_url: "https://api.dnsimple.test", access_token: "a1b2c3").services }
```

Comment:
In theory, services should also be accessible without authentication. In practice, I believe that this is currently not possible (same as #74).

This is mostly a FYI. I wonder if we should leave a note somewhere.